config/initializers/jwt_strategy.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'devise/strategies/authenticatable'
+
+module Devise
+  module Strategies
+    class JwtStrategy < Base
+      def valid?
+        token.present?
+      end
+
+      def authenticate!
+        payload = JwtService.decode token: token
```

Comment:
what will happen when token is invalid ? I think we need recuse from decode errors here and return unauthorized status